Procfile.dev

```diff
@@ -1,2 +1,2 @@
 web: rails s -p 3000
-client: sh -c 'rm app/assets/webpack/* || true && cd client && bundle exec rake react_on_rails:locale && npm run build:development'
+client: sh -c 'rm -rf public/webpack/development/* || true && cd client && bundle exec rake react_on_rails:locale && yarn run build:development'
```

Comment:
In the [orion-rails-layout project](https://github.cerner.com/orion/orion-rails-layouts/blob/master/Procfile.dev), they're using `npm`. Should we be using npm here or should we recommend the update to yarn? I'd like those to be consistent so we are close to what our consumers are doing.